.jenkins/job_beam_PreCommit_Website_Stage.groovy

```diff
@@ -0,0 +1,66 @@
+import common_job_properties
```

Comment:
My experience matches too.

Another alternative to consider: another Jenkins step before the DSL step that clones the other repository?